config/sites/now.uiowa.edu/config_split.patch.core.entity_view_display.node.article.default.yml

```diff
@@ -1,6 +1,83 @@
 adding:
+  dependencies:
+    config:
+      - field.field.node.article.field_article_type
+      - field.field.node.article.field_contact_reference
+      - field.field.node.article.field_embargo_information
+      - field.field.node.article.field_original_publication_date
+  third_party_settings:
+    layout_builder:
+      sections:
+        -
```

Comment:
I think we need to change the split type of this config to be full. Because we don't have UUID's for sections, there is no way to guarantee that this will work correctly. See https://www.drupal.org/project/drupal/issues/3208766#comment-14284826